<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListBooks</class>
 <widget class="QMainWindow" name="ListBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List of Books</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 127);
background-color: rgb(82, 176, 129);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_book">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 255);
background-color: rgb(85, 0, 127);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Add Book</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="go_back">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 255);
background-color: rgb(85, 0, 127);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_but">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 255);
background-color: rgb(85, 0, 127);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
